ivy_tests/test_ivy/test_frontends/test_tensorflow/test_metrics.py

```diff
@@ -566,3 +566,42 @@ def test_tensorflow_metrics_mean_squared_logarithmic_error(
         y_true=np.asarray(x[0], dtype=input_dtype[0]),
         y_pred=np.asarray(x[1], dtype=input_dtype[1]),
     )
+
+
+# Cosine Similarity
+@handle_cmd_line_args
+@given(
+    d_type=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        shared_dtype=True,
+        num_arrays=2
+    ),
+    y_true=helpers.array_values(
+        dtype=ivy.int32, shape=(1, 5), min_value=1, max_value=5
+    ),
+    y_pred=helpers.array_values(
+        dtype=ivy.int32, shape=(1, 5), min_value=5, max_value=10
+    ),

```

Comment:
I have initialized two separate arrays and removed all the unnecessary parameters. Is this correct @Aarsh2001?

Reply:
Here's how the results turned out:
![test](https://user-images.githubusercontent.com/64011471/192139350-a2e25fcb-de20-4ea3-97cb-d906724498d8.png)

Reply:
can you name your function to `test_tensorflow_cosine_similarity`, rest i feel the PR is good to go, thanks.

Reply:
Yeah, I changed it.